rust/rust.dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM janx/ubuntu-dev
+MAINTAINER Jan Keromnes "janx@linux.com"
+
+RUN sudo apt-get update -q \
+ && sudo apt-get upgrade -qy \
+ && sudo DEBIAN_FRONTEND=noninteractive apt-get install -qy \
+  build-essential \
+  cmake
+
+# Download Rust's source code.
+RUN git clone https://github.com/rust-lang/rust
+WORKDIR rust
+
+# Configure Cloud9 to use Rust's source directory as workspace (-w).
+RUN sudo sed -i "s/-w \/home\/user/-w \/home\/user\/rust/" /etc/supervisord.conf
+
+COPY config.toml .
```

Comment:
Nit: Please add a comment like `# Add default Rust build configuration.`. Also, could you please `wget` it from somewhere instead of manually importing it into this repo, if possible?